deploy/examples/radosnamesapce-mirrored.yaml

```
@@ -0,0 +1,22 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the RADOS namespace. If not set, the default is the name of the CR.
+  # name: namespace-a
+  # blockPoolName is the name of the CephBlockPool CR where the namespace will be created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: false
+    # remoteNamespace: ""
```

How about just show the default value so the example is clear?
```suggestion
    remoteNamespace: namespace-a
```

---

But it is only need to set if the namespace is not same